src/ucp/core/ucp_mm.c

```diff
@@ -1247,6 +1249,15 @@ void ucp_mem_print_info(const char *mem_spec, ucp_context_h context,
     fprintf(stream, "\n");
     fprintf(stream, "#\n");
 
+    status = ucp_rkey_pack(context, memh, &rkey_buffer, &rkey_size);
+    if (status != UCS_OK) {
+        fprintf(stream, "<Failed to pack rkey: %s>\n",
```

Comment:
in the rest of the function, we print information to stream and errors to stdout - why vice versa here?

Reply:
@Artemy-Mellanox fixed